float64 timestamp
int32   state
float64 distanceToWall
